modules/swift/examples/code_snippets/SampleCodeTest.swift

```diff
@@ -537,10 +585,10 @@ class SampleCodeTest {
                 SelectResult.expression(Expression.property("airline").from("route"))
             )
             .from(
-                DataSource.database(database!).as("airline")
+                DataSource.collection(collection).as("airline")
             )
             .join(
-                Join.join(DataSource.database(database!).as("route"))
+                Join.join(DataSource.collection(collection).as("route"))
```

Comment:
It's better to have two collections instead of just one for join examples such as airlines and routes collection.